src/smart-components/Thread/context/__test__/utils.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { UserMessage } from "@sendbird/chat/message";
+import { isParentMessage, isThreadMessage } from "../utils";
+
+const mockParentMessage = {
+  messageId: 1,
+  parentMessage: null,
+  parentMessageId: null,
+  threadInfo: {
+    lastRepliedAt: 100000,
+    replyCount: 1,
+    mostRepliedUsers: [],
+    updatedAt: 100000,
+  },
+};
+const mockThreadMessage = {
+  messageId: 2,
+  parentMessage: mockParentMessage,
+  parentMessageId: 1,
+  threadInfo: null,
+};
+
+describe('Thread/utils - isParentMessage', () => {
+  it('should comfirm if messages are parent message', () => {
+    expect(isParentMessage(mockParentMessage as UserMessage)).toBe(true);
+    expect(isParentMessage(mockThreadMessage as UserMessage)).toBe(false);
+  });
```

Comment:
true and false should be seperate assertions, same for `Thread/utils - isThreadMessage `

Reply:
That means,, it needs to be divided into two test?